FlutterRoadmap.md

```diff
@@ -15,7 +15,7 @@ by Oleksandr Leuschenko (@oleaxe)
 
 * **Language**
   * Dart (Must have)
-  * OOP (Nice to have)
+  * OOP (Must have)
```

Comment:
OOP principles will come naturally when junior devs work on patterns. I think that for the very beginning, it's rather "Nice to have" knowledge.